addons/mail/static/src/services/new_message_service/new_message_service.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+const { Component } = owl;
+
+export const newMessageService = {
+    dependencies: ["command"],
+    start(newEnv, { command }) {
+        command.add("Message a user", () => {
+            const env = Component.env;
+            const messagingMenu = env.messaging && env.messaging.messagingMenu;
+            if (!env.messaging.device.isMobile) {
+                env.messaging.chatWindowManager.openNewMessage();
+                messagingMenu.close();
+            } else {
+                messagingMenu.toggleMobileNewMessage();
+            }
+
+        }, {
+            category: "mail",
+            hotkey: "alt+shift+w",
```

Comment:
Just a note but that shortcut is almost impossible to use in azerty, with 3 keys that are right next to each other on the corner of the keyboard. But I don't really care as I'll always be using ctrl+k anyway.